### PR TITLE
Bookstore: add reducers and actions

### PR DESCRIPTION
**Pull request by obibaadoma**
Add Redux Toolkit (npm install react-redux @reduxjs/toolkit).
Create a directory that will contain all your Redux logic (/src/redux)
Configure a Redux store (/src/redux/store.js)

### CODE REVIEW

File: src/components/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+);
+
+// Reducer
+const booksReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOKS:
+      return {
+        ...state,
+        books: [...state.filter((book) => book.id !== action.id)],
+      };
```
Comment (V-Blaze) [inline src/components/redux/books/books.js]: - You have done great so far :+1: But there are a few requirements you need to complete before moving to the next milestone. :+1: :100: 
- [ ] You did well adding the action creators as required, but in your reducer, the action type of `ADD_BOOKS` seems to be removing a book and not adding, I'll suggest you rename or re-write this functionality correctly. :+1: :+1: :smile: 
- [ ] Also, in your Reducer, you are meant to handle the two action types `REMOVE_BOOKS` and `ADD_BOOKS`, but currently, you only have case for one action type, I'll suggest you add the other. :small_airplane: :wave:

File: src/components/redux/books/books.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+// define the action type
```
Comment (V-Blaze) [inline src/components/redux/books/books.js]: - [ ] Currently, your redux folder is inside the component folder which is not correct, its meant to be under `src` and not under components, I'll suggest you move the folder one level above. :smile: :+1:

File: README.md [hunk 3]
```diff
@@ -1 +1,154 @@
...
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (V-Blaze) [inline README.md]: - [ ]  I'll suggest you remove all these placeholder comments as they are there to help you fill out your readme. :+1:

File: README.md [hunk 4]
```diff
@@ -1 +1,154 @@
...
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [� License ](#-license-)
+
+
+
+# 📖 [Math-app] <a name="about-project"></a>
```
Comment (V-Blaze) [inline README.md]: - [ ] As required by MIcroverse, your readme file needs to be descriptive and look professional, I'll suggest you remove the name `Math App`,  and update the t=name correctly as this is the `Book-store ` project👍 👍

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @obibaadoma 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- INstalled redux tool kit✔
-  Clean and well-documented code👋 
-  Correct git flow✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/redux/books/books.js (hunk 2 above)
Comment (obibaadoma) [reply src/components/redux/books/books.js]: done, moved to source

File: README.md (hunk 3 above)
Comment (obibaadoma) [reply README.md]: Thanks, removed.

File: README.md (hunk 4 above)
Comment (obibaadoma) [reply README.md]: Great, thanks for the correction, necessary changes applied

File: src/components/redux/books/books.js (hunk 1 above)
Comment (obibaadoma) [reply src/components/redux/books/books.js]: Done, included remove case and edited add case.

**Review by christianbiring1 — APPROVED** [review]
Hi @obibaadoma , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- [ ] Great work on implementing the changes as required by the previous reviewer :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**